cime_config/cesm/config_archive.xml

```diff
@@ -3,7 +3,7 @@
   <comp_archive_spec compname="cam" compclass="atm">
     <rest_file_extension>\.[ri]\..*</rest_file_extension>
     <rest_file_extension>\.r[sh]\.*</rest_file_extension>
-    <hist_file_extension>\.h.*</hist_file_extension>
+    <hist_file_extension>\.h.*.nc$</hist_file_extension>
```

Comment:
Yeah,  we do use it. Would you mind mirroring these changes on the ACME side? I know we don't like to touche each others' config files. But for bug fixes like this I think it's appropriate.